e2e/tests/plugins/notebook/tags.e2e.spec.js

```diff
@@ -0,0 +1,123 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*
+This test suite is dedicated to tests which verify form functionality.
+*/
+
+const { test, expect } = require('@playwright/test');
+
+test.describe('Tagging in Notebooks', () => {
+    async function createNotebookAndEntry(page) {
```

Comment:
@scottbell can you move these functions out of the test describe and give them typing hints?

Reply:
e.g. https://github.com/nasa/openmct/blob/34442c53c6f34a88ff07a271577bbb039fae471c/e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js#L652-L657

Reply:
I can move the functions up, but I’m a little unclear why we need typing hints here as the functions are private test utilities.

Reply:
I approve of your optimism! 😀